.github/workflows/tests.yml

```
@@ -110,7 +109,47 @@ jobs:
             export EXTRA_OPTIONS=" "
           fi
 
-          python -m pytest $EXTRA_OPTIONS tests
+          python -m pytest $EXTRA_OPTIONS tests/runtime
+
+      - name: Run h2o benchmarking tests
+        continue-on-error: true
+        id: h2o_tests
+        env:
+            DASK_COILED__TOKEN: ${{ secrets.COILED_BENCHMARK_BOT_TOKEN }}
+            AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
+            AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}            
+        run: |
+          # Ensure we run additional tests when testing the latest coiled-runtime
+          if [[ ${{ matrix.runtime-version }} = 'latest' ]]
+          then
+            export EXTRA_OPTIONS="--run-latest"
+            export COILED_SOFTWARE_NAME=${{ env.COILED_SOFTWARE_NAME }}
+          else
+            export EXTRA_OPTIONS=" "
+          fi
+
+          python -m pytest $EXTRA_OPTIONS tests/h2o
```

Today there are only h2o benchmarks, but we'll soon be adding more (e.g. some parquet-related benchmarks). Given this, can make the directory structure / names use the more generic term "benchmarks" instead of "h2o benchmarks", etc.

```suggestion
          python -m pytest $EXTRA_OPTIONS tests/benchmarks
```

---

I've placed the h2o benchmarks in a separate subdirectory. /cc @MrPowers @rrpelgrim. The test directory now looks as follows:

```bash
$ tree tests/
tests/
├── __init__.py
├── __pycache__
├── benchmarks
│   ├── __init__.py
│   └── h2o
│       ├── __init__.py
│       └── test_h2o_benchmarks.py
├── runtime
│   ├── __init__.py
│   ├── test_build.py
│   └── test_coiled.py
├── stability
│   ├── __init__.py
│   ├── test_array.py
│   └── test_shuffle.py
└── test_deadlock.py
```